docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

```diff
@@ -164,6 +317,10 @@ automatically apply to application workloads. To configure applications to use t
 
    :::
 
+</TabItem>
+
+</Tabs>
+
 ## Validate
```

Comment:
Are these validation steps relevant for Appliance Mode Studio?

I didn't think anyone using it would have had the CanvOS repo downloaded?